SpecFlow.BDD.Selenium.POM.Automation.Mars/Automation.Mars.Test/TestCases/03 Skills.feature

```diff
@@ -0,0 +1,162 @@
+﻿Feature: 03 Skills
+
+As a Mars user, 
+I want to add，update, delete Skills which I have，
+So that traders can know my Skills.
+
+Background: Clean up skills
```

Comment:
same comments apply

Reply:
Thanks. I will implement data cleanup generated by tests of skills in new pull request.